shared/js/project.js

```diff
@@ -403,8 +426,16 @@ var projects = module.exports = {
           deferred.reject();
         } else {
           current = data;
-          projects.showProjectLevelHeader();
-          deferred.resolve();
+          if (data.migratedToS3) {
```

Comment:
This looks like it won't get hit on applab project levels. Can this be done for applab now so that we don't have to duplicate all the migration steps?